frontend/src/app/auth/auth.service.ts

```diff
@@ -16,10 +16,11 @@ export class AuthService {
   private _idToken: string;
   private _accessToken: string;
   private _expiresAt: number;
-  user: User;
+  private clientId: string = 't3sXyFtDUl0wFsHVsQsJbEa4en4bgPly';
```

Comment:
As a nice-to-have, moving this out of the codebase would be nice

Reply:
Why not just use environment variables? Cant angular inject them?